src/polar/Pie.tsx

```diff
@@ -87,6 +87,7 @@ interface PieProps extends PieDef {
   data?: any[];
   sectors?: PieSectorDataItem[];
   activeShape?: PieActiveShape;
+  inactiveShape?: PieActiveShape;
```

Comment:
nitty: Should we consider renaming the interface PieActiveShape to PieShape? Disregard if this would cause a breaking change though.